dora/shell/src/main/java/alluxio/cli/fs/command/SetTtlCommand.java

```diff
@@ -1,114 +0,0 @@
-/*
- * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
- * (the "License"). You may not use this work except in compliance with the License, which is
- * available at www.apache.org/licenses/LICENSE-2.0
- *
- * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
- * either express or implied, as more fully set forth in the License.
- *
- * See the NOTICE file distributed with this work for information regarding copyright ownership.
- */
-
-package alluxio.cli.fs.command;
-
-import alluxio.AlluxioURI;
-import alluxio.annotation.PublicApi;
-import alluxio.cli.CommandUtils;
-import alluxio.cli.fs.FileSystemShellUtils;
-import alluxio.client.file.FileSystemContext;
-import alluxio.exception.AlluxioException;
-import alluxio.exception.status.InvalidArgumentException;
-import alluxio.grpc.TtlAction;
-import alluxio.util.CommonUtils;
-
-import org.apache.commons.cli.CommandLine;
-import org.apache.commons.cli.Option;
-import org.apache.commons.cli.Options;
-
-import java.io.IOException;
-import javax.annotation.concurrent.ThreadSafe;
-
-/**
- * Sets a new TTL value for the file at path both of the TTL value and the path are specified by
- * args.
- */
-@ThreadSafe
-@PublicApi
-public final class SetTtlCommand extends AbstractFileSystemCommand {
```

Comment:
i think TTL still exists in Dora? Don't you want a command to manually override something? @huanghua78

Reply:
yesterday we had a discussion, most likely it is not valid in dora's env, and need to re-implement.